frontend/packages/knative-plugin/src/components/overview/__tests__/ConfigurationsOverviewList.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as utils from '@console/internal/components/utils';
+import ConfigurationsOverviewList from '../ConfigurationsOverviewList';
+import ConfigurationsOverviewListItem from '../ConfigurationsOverviewListItem';
+import { mockConfigurationsData } from '../__mocks__/overview-knative-mock';
+
+describe('ConfigurationsOverviewList', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(<ConfigurationsOverviewList configurations={mockConfigurationsData} />);
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should have title Configurations', () => {
+    const wrapper = shallow(<ConfigurationsOverviewList configurations={mockConfigurationsData} />);
+    expect(wrapper.find(utils.SidebarSectionHeading)).toHaveLength(1);
+    expect(
+      wrapper
+        .find(utils.SidebarSectionHeading)
+        .at(0)
+        .props().text,
+    ).toEqual('Configurations');
+  });
+
+  it('should render error Message when configurations array is empty', () => {
+    const wrapper = shallow(<ConfigurationsOverviewList configurations={[]} />);
+    const errorMsg = <span className="text-muted">No Configurations found for this resource.</span>;
+    expect(wrapper.contains(errorMsg)).toEqual(true);
```

Comment:
Let's omit checking for muted text and only for the actual data.
`expect(wrapper.text().includes('No Configurations found for this resource.')).toBe(true);`